.github/workflows/cd.yml

```
@@ -0,0 +1,41 @@
+name: cd
+
+on:
+  push:
```

Allow people to trigger it manually
```suggestion
  workflow_dispatch: 
  push:
```